### ISSUE TITLE
not able to get indices from elasticsearch

### ISSUE THREAD

**Issue by jameslee007** [KongZ/charts#40]
Hi,

Thanks for giving us this chart.  I am facing few problem to make the graylog up.

I have deployed the chart using helm3 in a kubernetes cluster. All pods are up and running. but graylog gives an error.
```

Caused by: org.elasticsearch.cluster.block.ClusterBlockException: blocked by: [SERVICE_UNAVAILABLE/1/state not recovered / initialized];
        at org.elasticsearch.cluster.block.ClusterBlocks.indicesBlockedException(ClusterBlocks.java:229) ~[elasticsearch-6.8.13.jar:6.8.13]
        at org.elasticsearch.action.admin.indices.alias.get.TransportGetAliasesAction.checkBlock(TransportGetAliasesAction.java:57) ~[elasticsearch-6.8.13.jar:6.8.13]
        at org.elasticsearch.action.admin.indices.alias.get.TransportGetAliasesAction.checkBlock(TransportGetAliasesAction.java:39) ~[elasticsearch-6.8.13.jar:6.8.13]
...... omitted
```

The elasticsearch cluster status is red and i am not able to view any indices and it shows below error.

{"error":{"root_cause":[{"type":"cluster_block_exception","reason":"blocked by: [SERVICE_UNAVAILABLE/1/state not recovered / initialized];"}],"type":"cluster_block_exception","reason":"blocked by: [SERVICE_UNAVAILABLE/1/state not recovered / initialized];"},"status":503}

Can you please help me how to solve this.

Thank you.

Regards,
James

**Comment by KongZ** [comment]
This is the error on Elasticsearch. How did you install the Elasticseach. From the error message, i guess the Elasticsearch could not write in data into disk or files corrupt.

**Comment by jameslee007** [comment]
Hi,

Thank you for your reply.

I have installed elasticsearch through the helm chart itself.

Let me check with the disk once and get back to you.

Regards,
James

**Comment by jameslee007** [comment]
I have separated the elasticsearch . now it is working. but having another issue regarding ingress. will comment on existing raised issue. As of now marking this issue as closed.
